src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -823,8 +823,10 @@ bool emberAfOperationalCredentialsClusterCSRRequestCallback(app::CommandHandler
             gFabricBeingCommissioned.GetOperationalKey()->Clear();
         }
 
-        keypair.Initialize();
-        SuccessOrExit(err = gFabricBeingCommissioned.SetOperationalKeypair(&keypair));
+        keypair = chip::Crypto::GetP256KeypairBuilder()->BuildP256KeyPairForOperationalKey(gFabricBeingCommissioned.GetFabricId());
```

Comment:
This should be FabricIndex, not fabric ID

Reply:
Updated.